traits/trait_types.py

```diff
@@ -3141,7 +3156,7 @@ class Map(TraitType):
 
         The following example defines a ``Person`` class::
 
-            >>> class Person(HasTraits):
+            >>>     erson(HasTraits):
```

Comment:
Something odd has happened to this line.

Reply:
Thanks; should be fixed in e2435a5626e50099aeb370994b8b91f5ac6506da.